CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 * Fixes
 
+## 2.0-patch.3
+
+* Fixes
+
+  * Reverted https://github.com/dcos/dcos-core-cli/pull/322
```

Comment:
A changelog targets users, here as a user I wouldn't understand this fix, even though I follow the PR link.

```suggestion
  * Fixed creation of nested groups, the regression was introduced in #322
```